standards/core/sync.md

```diff
@@ -28,13 +28,16 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 The protocol finds differences between two peers by
 comparing _fingerprints_ of _ranges_ of message _IDs_.
-_Ranges_ are encoded into payloads, exchanged between the peers and when the range _fingerprints_ are different, split into smaller (sub)ranges.
+_Ranges_ are encoded into payloads,
+exchanged between the peers and when
+the range _fingerprints_ are different, split into smaller (sub)ranges.
 This process repeats until _ranges_ include a small number of messages.
-At this point lists of message _IDs_ are sent for comparison instead of _fingerprints_ over entire ranges of messages.
+At this point lists of message _IDs_ are sent for comparison
+instead of _fingerprints_ over entire ranges of messages.
 
 #### Overview
-The `reconciliation` protocol follows the following heuristic:
-1. The requestor chooses a time range to sync.
+The `reconciliation` protocol operate on the following heuristic:
+1. The requestor chooses a sync range including time, cluster id and topics.
```

Comment:
Presumably, with "topics" here you refer to both pubsub topics and content topics? We should be explicit about this here and elsewhere, but I wonder if we keep the spec limited to pubsub topics (only)? Even if in implementation we add experimental content topics support, my guess is that we'll mature/test the pubsub topics implementation before validating that it works for content topics. At that point, the spec can be upgraded to `draft`, while a new raw version can include content topics. If you agree, the suggested change then is to be explicit about "pubsub topic" throughout.

Reply:
The mechanism is the same for pubsub or content topics but I agree about being explicit.

Reply:
> Presumably, with "topics" here you refer to both pubsub topics and content topics?

I agree being explicit is better.

This PR reflects the changes in this https://github.com/waku-org/nwaku/pull/3275
Both should be merged together ounce we have tested the current version (which include shard support).

Since both pubsub and content topic use the exact same mechanism testing separately doesn't make much sense.